modin/core/dataframe/pandas/partitioning/partition_manager.py

```diff
@@ -675,7 +676,7 @@ def map_partitions(
         NumPy array
             An array of partitions
         """
-        if np.prod(partitions.shape) <= 1.5 * CpuCount.get():
+        if not DynamicPartitioning.get():
```

Comment:
Do you think we should do that? Shouldn't the user be given more freedom to decide when to activate this option?
The user can activate this locally, only for the required operations.

Reply:
The user now has the ability to force the use of another code branch at his own choice, this is already more flexibility than before. And since this condition worked quite well before and considering that slowdowns are possible when using this new variable, I would replace the default behavior more carefully.

Reply:
Confused with similar code in `rebalance_partitions` function. Ok, leave it as is.